### DISCUSSION TITLE
請問是要如何使用多語言切換功能呢？

### DISCUSSION THREAD

**Discussion by RyanTokManMokMTM** [hugo-next/hugo-theme-next#147] [Q&A]
是只能在config設定一種語言嗎？
`defaultContentLanguage: zh-cn
languageCode: zh-CN`

**Comment by elkan1788** [comment]
主题可以支持多语言设计，目前支持：简体中文，繁体中文，英文和法语4种语言，如有其他语言需求，可以参考[i18n](https://github.com/hugo-next/hugo-theme-next/tree/main/i18n) 进行扩展即可。

关于多语言的切换，上面的配置只是说指定站点的默认语言，当你的文章有多种语言时，可以自动在站点主域名路径后加上语言名称进行切换，比如 https://xxx.com/en-US/ 之类，后续会在站点底层增加多语言切换的支持（现这里是使用google翻译，但是似乎现google翻译官方已经关闭，故无法支持当前主题使用）。
